coralproject/talk
# [next] Comments Count

## What does this PR do?
- Adds Comment Count

## How do I test this PR?
- Open the Stream
- Check the comment Count
- Write a comment
- Check the comment Count
